spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -5,6 +5,63 @@
 
   before { allow(Rails.application.config).to receive(:consider_all_requests_local) { false } }
 
+  describe 'GET /highlights' do
+    let!(:highlight1) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight2) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight3) { create(:highlight) }
+
+    let(:per_page) { 10 }
+    let(:query_params) do
+      {
+        source_type: 'openstax_page',
+        source_parent_ids: highlight1.source_parent_ids,
```

Comment:
should test with not all the source parent IDs as clients are likely to call with just one.